.github/workflows/integration_tests.yml

```
@@ -44,7 +44,7 @@ jobs:
           timestamp=$(date +'%Y%m%d%H%M')
           report_filename="${timestamp}_terraform_test_report.log"        
           
-          if ! make TESTARGS="2>&1" testacc > "$report_filename"; then
+          if ! make TESTARGS="" testacc > "$report_filename"; then
```

```suggestion
          if ! make testacc > "$report_filename"; then
```
Can we remove it completely like this?